spec/Review.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { shallow} from 'enzyme';
+import Review from '../client/src/components/Review.jsx';
+import { reviews, stars } from './testDummyData.js';
+
+describe('Review', () => {
+    it('should be defined', () => {
+      expect(Review).toBeDefined();
+    });
+    it('should render correctly', () => {
+      const tree = shallow(
```

Comment:
Another test you can look into is using Enzyme's find() method to look for specific html elements, and then checking for things with its props() method. For example, wrapper.find('#some-id').props().className could get the class of a specific element. The find() method could also select multiple elements, so using that in conjunction with Jest's .toHaveLength() method to check if a component has a certain number of some element might be useful to supplement your snapshot testing. 

I think that's how those work, but as always, refer to Enzyme and Jest's docs for clarification.